job-shop-scheduling/test_job_shop_scheduler.py

```diff
@@ -385,6 +386,30 @@ def test_get_jss_bqm(self):
         bqm = get_jss_bqm(jobs, max_time)
         self.assertIsInstance(bqm, BinaryQuadraticModel)
 
+    def test_stitch_kwargs(self):
+        """ Ensure stitch_kwargs is being passed through get_jss_bqm to dwavebinarycsp.stitch
+        """
+        jobs = {"sandwich": [("bread", 1), ("roast_beef", 1)],
+                "french_toast": [("egg", 1), ("bread", 1)]}
+        max_time = 3
+        impossible_stitch_kwargs = {"max_graph_size": 0}
+
+        # ImpossibleBQM should be raised, as the max_graph size is too small
+        self.assertRaises(ImpossibleBQM, get_jss_bqm, jobs, max_time, impossible_stitch_kwargs)
+
+
+class TestGetBqm(unittest.TestCase):
+    def test_stitch_kwargs(self):
```

Comment:
You're right, TestGetJssBqm does make sure the args are passed to get_bqm and then to dwavebinarycsp.stitch. However since get_jss_bqm doesn't directly call dwavebinarycsp.stitch I thought it might be more "appropriate" to test get_bqm by itself. 

Do you think it would just be best to remove the TestGetBqm class altogether then?

Reply:
Could you please clarify a bit more? I don't see the code making any direct calls to `get_bqm`.

From what I can see, the unit test `test_stitch_kwargs(..)` in both classes are identical. Is that correct?

Reply:
Wow, really sorry about that, check now.

I assumed that you meant TestGetBqm was unnecessary since TestGetJssBqm calls get_bqm, and if TestGetJssBqm passed, then that means get_bqm also works. My comment should now make sense in the fixed test.

Reply:
Saw the fix. Looks good.